syntax = "proto3";

package cakelab.arflow_grpc.v1;

import "cakelab/arflow_grpc/v1/device.proto";
import "cakelab/arflow_grpc/v1/session.proto";

option csharp_namespace = "CakeLab.ARFlow.Grpc.V1";

message CreateSessionRequest {
  SessionMetadata session_metadata = 1;
  Device device = 2;
}
